war/src/main/java/io/lumeer/engine/controller/OrganizationFacade.java

```diff
@@ -127,7 +128,7 @@ public Organization readOrganization(final String organizationCode) {
     *       organization data
     * @return id of the organization
     */
-   public String createOrganization(final Organization organization) {
+   public String createOrganization(final Organization organization) throws InvalidDocumentKeyException {
       String id = dataStorage.createDocument(LumeerConst.Organization.COLLECTION_NAME, organization.toDataDocument());
       databaseInitializer.onOrganizationCreated(id);
       securityFacade.addOrganizationUserRole(organization.getCode(), userFacade.getUserEmail(), LumeerConst.Security.ROLE_MANAGE);
```

Comment:
What do you mean by move this line? It is a line with the declaration of a method.

But finally, I see that data are not dependent on input so I suggest to do:

`try {`
`   securityFacade.addOrganizationUserRole(organization.getCode(), userFacade.getUserEmail(), LumeerConst.Security.ROLE_MANAGE);`
`} catch (InvalidDocumentKeyException e) {`
`   // Never can happen`
`}`